alexandria/math/src/signed_integers.cairo

```diff
@@ -800,6 +802,8 @@ fn i17_min(lhs: i17, rhs: i17) -> i17 {
 
 // ====================== INT 33 ======================
 
+use serde::U32Serde;
```

Comment:
This could move to the top of the file:
`use serde::{Serde, BoolSerde, U32Serde};`